packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -50,21 +53,29 @@ export const HoldPane = (): JSX.Element => {
     };
   }, [startTime]);
 
+  const showSpinner = (): JSX.Element => {
```

Comment:
nit: semantically this is the spinner itself as opposed to a function that triggers a spinner to show
```suggestion
  const ResumeSpinner = (): JSX.Element => {
```

then where its used:
```
{resumingCall && <ResumeSpinner  />}
```